docs/jupiterone.md

```diff
@@ -96,21 +96,30 @@ https://github.com/JupiterOne/sdk/blob/main/docs/integrations/development.md
 
 The following entities are created:
 
-| Resources | Entity `_type` | Entity `_class` |
-| --------- | -------------- | --------------- |
-| Account   | `acme_account` | `Account`       |
-| User      | `acme_user`    | `User`          |
-| UserGroup | `acme_group`   | `UserGroup`     |
+| Resources          | Entity `_type`             | Entity `_class` |
+| ------------------ | -------------------------- | --------------- |
+| Account            | `asana_account`            | `Account`       |
+| Project            | `asana_project`            | `Project`       |
+| Project Membership | `asana_project_membership` | `AccessRole`    |
+| Team               | `asana_team`               | `Team`          |
+| User               | `asana_user`               | `User`          |
+| Workspace          | `asana_workspace`          | `Organization`  |
 
 ### Relationships
 
 The following relationships are created:
 
-| Source Entity `_type` | Relationship `_class` | Target Entity `_type` |
-| --------------------- | --------------------- | --------------------- |
-| `acme_account`        | **HAS**               | `acme_group`          |
-| `acme_account`        | **HAS**               | `acme_user`           |
-| `acme_group`          | **HAS**               | `acme_user`           |
+| Source Entity `_type`      | Relationship `_class` | Target Entity `_type`      |
+| -------------------------- | --------------------- | -------------------------- |
+| `asana_account`            | **HAS**               | `asana_workspace`          |
+| `asana_project`            | **HAS**               | `asana_project_membership` |
+| `asana_project_membership` | **HAS**               | `asana_user`               |
+| `asana_team`               | **HAS**               | `asana_project`            |
```

Comment:
I'm also thinking if:

`asana_team` -ASSIGNED-> `asana_project`

would be appropriate here. In plain English it does seem as if some team is assigned to particular project and isn't necessarily parent of it.

Reply:
I agree also, it makes more sense that way.